commcare_connect/opportunity/views.py

```diff
@@ -1393,3 +1395,109 @@ def sync_deliver_units(request, org_slug, opp_id):
 
     return HttpResponse(content=message, status=status)
 
+
+def user_visit_verification(request, org_slug, opp_id, pk):
```

Comment:
We will need functionally correct perms etc. Whether its' org_member_required or other checks